src/data/partners.ts

```diff
@@ -0,0 +1,165 @@
+// Todo: Fill with Partners links & images (logos)
```

Comment:
We need the missing data for Partners links & images.
How do we handle lack of them? Not every partner might have some logo or website